src/warnet/control.py

```diff
@@ -232,52 +244,61 @@ def run(scenario_file: str, additional_args: tuple[str]):
         print(f"Failed to start scenario: {scenario_name}")
         print(f"Error: {e.stderr}")
 
+    if debug:
+        print("Waiting for commander pod to start...")
+        wait_for_pod(name)
+        _logs(pod_name=name, follow=True)
+        print("Deleting pod...")
+        delete_pod(name)
+
 
 @click.command()
 @click.argument("pod_name", type=str, default="")
 @click.option("--follow", "-f", is_flag=True, default=False, help="Follow logs")
 def logs(pod_name: str, follow: bool):
     """Show the logs of a pod"""
-    follow_flag = "--follow" if follow else ""
+    return _logs(pod_name, follow)
+
+
+def _logs(pod_name: str, follow: bool):
+    """Show the logs of a pod"""
     namespace = get_default_namespace()
 
-    if pod_name:
+    if pod_name == "":
         try:
-            command = f"kubectl logs pod/{pod_name} -n {namespace} {follow_flag}"
-            stream_command(command)
-            return
+            pods = get_pods()
+            pod_list = [item.metadata.name for item in pods.items]
         except Exception as e:
-            print(f"Could not find the pod {pod_name}: {e}")
+            print(f"Could not fetch any pods in namespace {namespace}: {e}")
+            return
 
-    try:
-        pods = run_command(f"kubectl get pods -n {namespace} -o json")
-        pods = json.loads(pods)
-        pod_list = [item["metadata"]["name"] for item in pods["items"]]
-    except Exception as e:
-        print(f"Could not fetch any pods in namespace {namespace}: {e}")
-        return
+        if not pod_list:
+            print(f"Could not fetch any pods in namespace {namespace}")
+            return
 
-    if not pod_list:
-        print(f"Could not fetch any pods in namespace {namespace}")
-        return
+        q = [
+            inquirer.List(
+                name="pod",
+                message="Please choose a pod",
+                choices=pod_list,
+            )
+        ]
+        selected = inquirer.prompt(q, theme=GreenPassion())
+        if selected:
+            pod_name = selected["pod"]
+        else:
+            return  # cancelled by user
 
-    q = [
-        inquirer.List(
-            name="pod",
-            message="Please choose a pod",
-            choices=pod_list,
-        )
-    ]
-    selected = inquirer.prompt(q, theme=GreenPassion())
-    if selected:
-        pod_name = selected["pod"]
-        try:
-            command = f"kubectl logs pod/{pod_name} -n {namespace} {follow_flag}"
-            stream_command(command)
-        except Exception as e:
-            print(f"Please consider waiting for the pod to become available. Encountered: {e}")
-    else:
-        pass  # cancelled by user
+    container_name = "bitcoincore" if pod_name.startswith("tank") else None
```

Comment:
we can't ever rely on tank names conforming to any kind of convention

Reply:
fixed up here: https://github.com/bitcoin-dev-project/warnet/pull/614/commits/04be184b91f8d3bf370532add51e80d47e28bd2f